### PR TITLE
Update label and add spinner for pending migration card

### LINKED ISSUE #165: Migration Cards should start with a pending label
Currently, there's a lag once migration starts, and the cards are not updating immediately.  When a migration is initiated but not yet started, show the progress card with a "Pending" label.

### PR DESCRIPTION
**Pull request by mturley**
Closes #165 (p1).

screen shot 2018-04-24 at 3 49 19 pm

### CODE REVIEW

File: app/javascript/common/API.js [hunk 1]
```diff
@@ -35,4 +35,4 @@ export default {
   }
 };
 
-export const globalMockMode = false;
+export const globalMockMode = true;
```
Comment (michaelkro) [inline app/javascript/common/API.js]: Makes tests unhappy

Comment (mturley) [reply app/javascript/common/API.js]: oh no! good catch.

**Comment by mturley** [comment]
Passes `yarn test` locally now, sorry for the false start.

**Comment by priley86** [comment]
thanks!

**Pull request merged**